src/main/java/org/apache/commons/lang3/time/StopWatch.java

```diff
@@ -671,4 +680,368 @@ public void unsplit() {
+     *
+     * @since 3.16
+     */
+    public <T> Consumer<T> accept(Consumer<T> consumer) {
+        return (arg) -> {
+            if (isStopped()) {
+                start();
+            } else if (isSuspended()) {
+                resume();
+            }
+            try {
+                consumer.accept(arg);
+            } finally {
+                suspend();
+            }
+        };
+    }
+
+    /**
+     * Take the time of the execution of a given {@linkplain BiConsumer}.
+     *
+     * <p>
+     * <b>Take the time of given {@linkplain BiConsumer}</b>
+     * <pre>{@code
+     * final StopWatch watch = StopWatch.create();
+     *
+     * watch.accept((first, second) -> process(first, second)).accept("A", "B");
+     * }</pre>
+     * </p>
+     * <p>
+     *
+     *  @param consumer the consumer those application should be measured
+     *
+     * @return the given consumer prepared to take time if applied
+     *
+     * @throws IllegalStateException if the StopWatch is not stopped or suspended
+     *
+     * @param <T> the type of the first argument to the consumer
+     * @param <U> the type of the second argument to the consumer
+     *
+     * @since 3.16
+     */
+    public <T, U> BiConsumer<T, U> accept(BiConsumer<T, U> consumer) {
+        return (first, second) -> {
+            if (isStopped()) {
+                start();
+            } else if (isSuspended()) {
+                resume();
+            }
+            try {
+                consumer.accept(first, second);
+            } finally {
+                suspend();
+            }
+        };
+    }
+
+    /**
+     * Take the time of the execution of a given {@linkplain TriConsumer}.
+     *
+     * <p>
+     * <b>Take the time of given {@linkplain TriConsumer}</b>
+     * <pre>{@code
+     * final StopWatch watch = StopWatch.create();
+     *
+     * watch.accept((first, second, third) -> process(first, second, third)).accept("A", "B", "C");
+     * }</pre>
+     * </p>
+     * <p>
+     *
+     *  @param consumer the consumer those application should be measured
+     *
+     * @return the given consumer prepared to take time if applied
+     *
+     * @throws IllegalStateException if the StopWatch is not stopped or suspended
+     *
+     * @param <T> the type of the first argument to the consumer
+     * @param <U> the type of the second argument to the consumer
+     *
+     * @since 3.16
+     */
+    public <T, U, V> TriConsumer<T, U, V> accept(TriConsumer<T, U, V> consumer) {
+        return (firstArg, secondArg, thirdArg) -> {
+            if (isStopped()) {
+                start();
+            } else if (isSuspended()) {
+                resume();
+            }
+            try {
+                consumer.accept(firstArg, secondArg, thirdArg);
+            } finally {
+                suspend();
+            }
+        };
+    }
+
+
+    /**
+     * Take the time of the execution of a given {@linkplain Predicate}.
```

Comment:
Use `@link` like the rest of the code base.

Reply:
s/linkplain/link/g